style.css

```diff
@@ -7,3 +8,94 @@
 html {
   font-size: 62.5%;
 }
+
+body {
+  font-family: "Inter", sans-serif;
+  line-height: 1;
+  background-color: #e5e5e5;
+}
+
+header,
+main {
+  width: 37.5rem;
+  margin: 0 auto;
+}
+
+.bg-color {
+  background-color: #fff;
+}
+
+.container {
+  padding: 0 2.7rem;
+  margin: 0 auto;
+}
+
+/* HEADER */
+.main-header {
+  text-align: center;
+}
+
+.main-nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 4.8rem;
+  padding: 3rem 0;
+}
+
+.menu-icon {
+  width: 4rem;
+  height: 4rem;
+}
+
+.menu-icon:hover {
+  cursor: pointer;
+}
+
+/* HERO SECTION */
+.section-hero {
+  background-image: url(imgs/header-illsutration-mobile@2x.png);
```

Comment:
- [ ] **Good job so far 👏 👏** However, as suggested by the previous reviewer it would be better to align the content should be aligned with **flexbox**. This gives us a lot of options to align all our elements, plus, it's responsive. 👍 👍 
you may kindly follow this [link ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)for more info

Reply:
![Capture](https://user-images.githubusercontent.com/131291114/234898901-491e9b38-08ed-4157-91a2-77df621adb16.PNG)

Reply:
Hi @TuyishimireEric .

Do you think it should be like the following picture.